packages/webembeds-core/src/utils/html.utils.ts

```diff
@@ -121,21 +121,25 @@ async function uploadImageByUrl(url: string) {
 
   const { data, errors } = await fetchGraphQL({
     query: `
-      mutation UploadImageByURL($url: ImageUrl!) {
-        uploadImageByURL(url: $url)
+      mutation UploadImageByURLV2($input: UploadImageInput!) {
+        uploadImageByURLV2(input: $input) {
```

Comment:
Just use `uploadImageByURL` with the same signature. We plan to remove V2 at some point.